spec/deployment_spec.rb

```diff
@@ -114,3 +116,203 @@
     end
   end
 end
+
+if Config.longhorn_enabled
```

Comment:
Can you separate this out into its own spec file? `longhorn_spec.rb` or similar?